### PR TITLE
Add CLI debug and verbose as user settable arguments

### PR DESCRIPTION
**Pull request by brianlball**
for issue:
https://github.com/NREL/OpenStudio-server/issues/526
in coordination with:
https://github.com/NREL/OpenStudio-PAT/pull/123

tested on PAT example projects and through CLI.

default value is on for both since i think this is needed to get some of the results for some of the CI tests.  If not, then the default should be off for performance, which is significant.

traps invalid values and uses --debug and --verbose as defaults in that case, per below:

[22:55:29.780544 INFO] Directory is /mnt/openstudio/analysis_63071db8-3071-4b67-9554-edfe165b8c58/data_point_90156ecd-59ae-4669-bc3d-39cc90575e32
[22:55:29.780602 INFO] Opening run.log file '/mnt/openstudio/analysis_63071db8-3071-4b67-9554-edfe165b8c58/data_point_90156ecd-59ae-4669-bc3d-39cc90575e32/run/run.log'
[22:55:29.780660 WARN] CLI_Debug option: fake is not valid.  Using --debug instead.
[22:55:29.780810 WARN] CLI_Verbose option: fake is not valid.  Using --verbose instead.

### CODE REVIEW

File: local_setup_scripts/win64/nuke.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash -e
```
Comment (nllong) [inline local_setup_scripts/win64/nuke.sh]: Did you mean to commit all these files? If so, then I think we need a README.md in the root of this folder that describes what they all do.

File: server/app/jobs/dj_jobs/run_simulate_data_point.rb [hunk 2]
```diff
@@ -185,14 +185,22 @@ def perform
       @sim_logger.info "Directory is #{simulation_dir}"
       run_log_file = File.join(run_dir, 'run.log')
       @sim_logger.info "Opening run.log file '#{run_log_file}'"
-
+      #add check for valid CLI option or ""
+	    unless ["","--debug"].include?(@data_point.analysis.cli_debug)
```
Comment (nllong) [inline server/app/jobs/dj_jobs/run_simulate_data_point.rb]: looks good, can you make sure to move this indent back inline?

File: local_setup_scripts/win64/nuke.sh (hunk 1 above)
Comment (brianlball) [reply local_setup_scripts/win64/nuke.sh]: yeah, sounds good

File: server/app/jobs/dj_jobs/run_simulate_data_point.rb (hunk 2 above)
Comment (brianlball) [reply server/app/jobs/dj_jobs/run_simulate_data_point.rb]: yep, hidden tab in there. removed it

**Pull request merged**